DEVELOPMENT.md

```diff
@@ -1,330 +1,96 @@
 
-Wherever possible, dates should be formatted as `YYYY-MM-DD` so as to be
-directly compatible with `xsd:date`. Static Dates should be used in generated
-data so as to avoid unnecessary changes in the repository on build.
+When submitting a pull request, the proposed changes must pass the Continuous Integration tests.
+These tests can be run in the local developer environment by running `npm test` from the root
+of the repository. This will run the test located in the `/packages/traceability-schemas/__tests__/`
+folder.
 
-### Place (as an example)
+Common errors for not passing these tests include:
 
-- [JSON Schema](./docs/schemas/Place.json)
-- [Data Generator](./packages/traceability-schemas/src/generators/Place.js)
-- [JSON-LD Context (derived)](./docs/contexts/traceability-v1.jsonld)
-- [Vocabulary Definition (derived)](https://w3id.org/traceability#place)
+1. Malformed incorrect yml indentation
+2. Incorrect OpenAPI yml syntax
+3. Broken proofs as a result of context changes
+4. Example JSON files that don't pass schema validation
+5. Cascading changes as a result of changing a `common` RDF class used by multiple files
 
-### Common Environment Variables for Build and Development
+## Continuous Deployment
 
-Some useful environment variables to make testing, generation, and validation
-stricter or more verbose are detailed here:
+After a pull request is merged, the Continuous Delivery (CD) script will create the spec document
+which is deployed to [https://w3c-ccg.github.io/traceability-vocab/](https://w3c-ccg.github.io/traceability-vocab/).
+A summary of scripts which produce this document are described as follows.
 
-```
-VERBOSE_BUILD=true        # sets general verbosity levels up when building
-
-FULL_ERROR_HANDLING=true  # forces hard stops at points in the process
-                         # when warning or other exceptions are caught
+1. `schemas-to-openapi.js` builds the yml file for the [Open API Spec](https://w3c-ccg.github.io/traceability-vocab/openapi/)
+2. `openapi-to-context.js` builds the JSON-LD context from the Open API spec which is written to `/docs/contexts/traceability-v1.jsonld`
+3. `schemas-to-vocab.js` builds the html files for the schemas which are referenced from `/docs/index.html` in the spec
 
-VERBOSE_BUILD_GENERAL     # increases verbosity in object generation on
-                         # common objects
+## Generating Proofs
 
-VERBOSE_BUILD_AG=true     # industry vertical specific stops, see also
-                         # VERBOSE_BUILD_STEEL and related
-```
+New credentials can be added to the vocabulary by adding a new file inside `/docs/openapi/components/schemas/credentials/`
+with an issuer `did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn`. From there you can goto the
+`/packages/traceability-schemas/scripts/` directory and run `node regenerate.js`. This will generate a proof for credentials
+without a proof or otherwise fail to validate because of a broken proof.
```

Comment:
```suggestion
with issuer `did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn`. From there you can go to the
`/packages/traceability-schemas/scripts/` directory and run `node regenerate.js`. This will generate a proof for credentials
without a proof, or fail to validate because of a broken proof.
```